layouts/partials/social-icons.html

```diff
@@ -1,10 +1,10 @@
 {{ with .Site.Social.instagram -}}
-<a class="py-2 px-2 d-none d-lg-inline-block" href="https://twitter.com/{{ . }}"><i class="fab fa-twitter"></i></a>
+<a class="py-2 px-2 d-inline-block" href="https://twitter.com/{{ . }}"><i class="fab fa-twitter"></i></a>
```

Comment:
Thanks for spotting this! I gave these changes a test, and it will work. It can be simplified further by removing "d-inline-block", which has no effect in this context.